src/patternfly/components/LoginBox/login-box.scss

```diff
@@ -62,4 +62,13 @@
   padding-right: var(--pf-c-login-box__footer--PaddingRight);
   padding-bottom: var(--pf-c-login-box__footer--PaddingBottom);
   padding-left: var(--pf-c-login-box__footer--PaddingLeft);
+
+  // Hover behavior
+  & a:hover {
```

Comment:
is the `&` necessary? We also don't typically style elements themselves... we would create a BEM element out of it and style that class.